code/modules/events/hostile_infestation.dm

```diff
@@ -0,0 +1,125 @@
+//Hostile Infestation: A variant of the infestation event that spawns small numbers of hostile mobs.
+//Currently only fires if there are 2 or more security officers
+
+#define LOC_KITCHEN 0
```

Comment:
Don't really have any other issues with this PR but I really don't know why you made these defines, they don't actually do anything. And the pick(all of these numbers) could just as easily be done by rand(1,8).